server/form-pages/apply/basic-information/placementDate.ts

```diff
@@ -0,0 +1,49 @@
+import type { TasklistPage, ObjectWithDateParts } from 'approved-premises'
+import { dateAndTimeInputsAreValidDates } from '../../../utils/utils'
+
+export default class PlacementDate implements TasklistPage {
+  name = 'placement-date'
+
+  title = 'Is [release-date] the date you want the placement to start?'
```

Comment:
I guess this will be hooked up later?

Reply:
Yeah, we need access to the session here